controllers/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1923,4 +1979,67 @@ var (
   spec:
     timeoutSeconds: 28800 # 8 hours
   `
+
+	spaceAdmin test.TemplateObject = `
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: space-admin
+    namespace: ${NAMESPACE}
+  rules:
+    # examples
+    - apiGroups:
+        - ""
+      resources:
+        - "secrets"
+        - "serviceaccounts"
+      verbs:
+        - get
+        - list
+  `
+	spaceAdminRb test.TemplateObject = `
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: ${USERNAME}-space-admin
+    namespace: ${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: space-admin
+  subjects:
+    - kind: User
+      name: ${USERNAME}
+`
+	spaceViewer test.TemplateObject = `
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: space-viewer
+    namespace: ${NAMESPACE}
+  rules:
+    # examples
+    - apiGroups:
+        - ""
+      resources:
+        - "secrets"
+        - "serviceaccounts"
+      verbs:
+        - get
+        - list
```

Comment:
I know that it should have no effect on the result of the tests, but the `space-viewer` role has the same content as `space-admin` - it would make more sense if they were different

Reply:
sure, but indeed, it doesn't affect the tests

Reply:
done in 0ca2085